### PR TITLE
Update classifiers

### PR DESCRIPTION
**Pull request by hugovk**
Also, now 2.6 is gone, replace function calls with set literals.

And add fix a typo in an `assertIn` which used to be an `assertTrue`.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -33,6 +33,15 @@
           "Operating System :: Microsoft :: Windows",
           "Operating System :: POSIX",
           "Programming Language :: Python",
+          "Programming Language :: Python :: 2",
+          "Programming Language :: Python :: 2.7",
+          "Programming Language :: Python :: 3",
+          "Programming Language :: Python :: 3.3",
```
Comment (iMichka) [inline setup.py]: I juste removed 3.3 with 1ed25bda6a25f7f1bd4efdf9e6ed64c43afe592f

Could you remove it from here too?

**Comment by iMichka** [comment]
Hi. Cool cleanup here. Really appreciated. I made a small comment. Once the change is made and the tests are fine, I'll merge this.

File: setup.py (hunk 1 above)
Comment (hugovk) [reply setup.py]: Removed!

**Comment by hugovk** [comment]
Travis are running a bit behind at the moment, so patience needed :) https://www.traviscistatus.com/incidents/4vdl52d28hz3

**Comment by hugovk** [comment]
@iMichka How does this look?

**Comment by iMichka** [comment]
Hi. Sorry it took me so long ... I was busy with other projects. I hope I can finish the 2.0.0 release during the summer, so that the latest changes are released. Thanks for your contribution!

**Pull request merged**